tensorflow/lite/micro/kernels/kernel_util.h

```diff
@@ -31,16 +31,17 @@ namespace micro {
 // The Flexbuffer library is inline heavy, which causes code bloat when
 // custom ops are used. Wrapping with a function is a portable way to avoid
 // this bloat
-const flexbuffers::Map FlexbuffersWrapperGetRootAsMap(const uint8_t* buffer,
-                                                      size_t size);
-
-int32_t FlexbuffersWrapperAsInt32(const flexbuffers::Map& m, const char* key);
-
-bool FlexbuffersWrapperAsBool(const flexbuffers::Map& m, const char* key);
-
-float FlexbuffersWrapperAsFloat(const flexbuffers::Map& m, const char* key);
-
-bool FlexbuffersWrapperIsNull(const flexbuffers::Map& m, const char* key);
+class LiteVector : public flexbuffers::Vector {
```

Comment:
nit: can we use a more descriptive name than LiteVector?

Maybe FlexbufferVectorWrapper?

Reply:
why do we need to inherit from flexbuffers::Vector?

Reply:
We need access to   
const uint8_t *data_;
uint8_t byte_width_;
Both are protected members in Object, which is Vector's ancestor class.

Reply:
sgtm re. naming.